isis/src/base/objs/Angle/Angle.cpp

```diff
@@ -225,7 +225,8 @@ namespace Isis {
       IString msg = "Cannot compare a invalid angles with the < operator";
       throw IException(IException::Programmer, msg, _FILEINFO_);
     }
-
+    // != comparison allows for angles that are considered equal to be treated
+    // as being equal. == operator override uses qFuzzyCompare().
```

Comment:
I am confused by this. Is this saying that the code makes != equal == in some cases?

Reply:
never mind - I see what this is doing...